code/game/gamemodes/clock_cult/clockcult.dm

```diff
@@ -171,7 +171,7 @@ GLOBAL_VAR(clockcult_eminence)
 
 /proc/generate_clockcult_scriptures()
 	//Generate scriptures
-	for(var/categorypath in typesof(/datum/clockcult/scripture))
+	for(var/categorypath in subtypesof(/datum/clockcult/scripture))
```

Comment:
why would the root scripture ever need to be in the global list of scriptures?

Reply:
`subtypesof` is `typesof` minus the root type.

Reply:
oh right yea